tools/shared.py

```diff
@@ -812,9 +719,9 @@ def exe_suffix(cmd):
 LLVM_COMPILER=os.path.expanduser(build_llvm_tool_path(exe_suffix('llc')))
 
 EMSCRIPTEN = path_from_root('emscripten.py')
-EMCC = path_from_root('emcc')
-EMXX = path_from_root('em++')
-EMAR = path_from_root('emar')
+EMCC = path_from_root('emcc.py')
```

Comment:
why, and why these three (and not emranlib just below etc.)?

Reply:
1. Calling emcc recursively will cause multiple duplicated Python version warnings
2. There are many tests that call `[PYTHON, EMCC]` and expect empty `stderr` output. A Python version warning will break all of them.

The version warning occurs only on `emcc` wrapper so this change will remove the problems.

emranlib is just a no-op, em-config already allows Python 3, so no need to change. (Do we want it to require Python 2?)